<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="server_port_lineEdit"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QTextEdit" name="send_textEdit"/>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="send_Button">
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="server_port_label">
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="server_address_lineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="server_address_label">
     <property name="text">
      <string>服务器地址</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="connect_Button">
     <property name="text">
      <string>连接网络</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5" colspan="2">
    <widget class="QTextBrowser" name="receive_textBrowser"/>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="clear_receive_Button">
     <property name="text">
      <string>清空接收数据</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
